public/static/docs/user-guide/privacy.md

```diff
@@ -1,13 +1,12 @@
 # DVC privacy policy to access Google Services and APIs
 
 DVC is using the [Google Drive API](https://developers.google.com/drive/) to
-make it easier to store and version files in the Google Drive storage. For the
-details on how to setup it check the respective sections in the `dvc remote add`
-and `dvc remote modify` commands as well as the relevant Google Drive section of
-this User Guide.
+make it easier to store files on Google Drive. For more details on how to set it
+up, refer to
+[Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote)
 
-DVC uses the _DVC_ application on Google Cloud Platform. That is the name you
-will see in a consent screen. DVC integration is governed by common policies
+DVC uses the _DVC_ Project on Google Cloud Platform. (That is the name you will
```

Comment:
why is it upper case, btw?

Reply:
It's Application now BTW. Upper case because it's a special kind of app (a GCP app, used to setup GAPI integrations) vs the general concept of software applications. The distinction is useful, for example, in the **Usage in other packages or applications** section.

Reply:
👍